### PR TITLE
Use migration schema for norm abbreviation

### PR DESCRIPTION
**Pull request by HPrinz**
RISDEV-2517

### CODE REVIEW

File: backend/src/test/java/de/bund/digitalservice/ris/caselaw/domain/LookupTableImporterServiceTest.java [hunk 1]
```diff
@@ -49,9 +47,12 @@ class LookupTableImporterServiceTest {
...
   @SpyBean private LookupTableImporterService service;
 
-  @MockBean private DatabaseDocumentTypeRepository databaseDocumentTypeRepository;
+  @MockBean
+  private de.bund.digitalservice.ris.caselaw.adapter.database.r2dbc.lookuptable
+          .DatabaseDocumentTypeRepository
+      databaseDocumentTypeRepository;
 
-  @MockBean private JPADocumentTypeRepository jpaDocumentTypeRepository;
+  @MockBean private JPADocumentTypeRepository JPADocumentTypeRepository;
```
Comment (rvp-c) [inline backend/src/test/java/de/bund/digitalservice/ris/caselaw/domain/LookupTableImporterServiceTest.java:55]: Variable/field names start with a lowercase letter. In this case I'm total fine with the old version. or jPADocumentTypeRepository

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java [hunk 2]
```diff
@@ -16,30 +16,18 @@ public NormAbbreviationService(NormAbbreviationRepository repository) {
...
-    if (page != null && size != null) {
-      pageOffset = page * size;
-    }
-
-    return repository.findBySearchQuery(query, size, pageOffset);
+    return repository.findBySearchQuery(query, size, page);
   }
 
   public Mono<List<NormAbbreviation>> getNormAbbreviationByAwesomeSearchQuery(
       String query, Integer size, Integer page) {
```
Comment (rvp-c) [inline backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java:29]: why we don't need the page anymore? if we don't needed please also remove it from the method params.

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java [hunk 3]
```diff
@@ -16,30 +16,18 @@ public NormAbbreviationService(NormAbbreviationRepository repository) {
     this.repository = repository;
   }
 
-  public Mono<NormAbbreviation> getNormAbbreviationById(UUID uuid) {
+  public NormAbbreviation getNormAbbreviationById(UUID uuid) {
     return repository.findById(uuid);
   }
 
   public Flux<NormAbbreviation> getNormAbbreviationBySearchQuery(
```
Comment (rvp-c) [inline backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java:23]: can we remove the flux here?

File: backend/src/test/java/de/bund/digitalservice/ris/caselaw/domain/LookupTableImporterServiceTest.java (hunk 1 above)
Comment (HPrinz) [reply backend/src/test/java/de/bund/digitalservice/ris/caselaw/domain/LookupTableImporterServiceTest.java]: Solved 👍

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java (hunk 3 above)
Comment (HPrinz) [reply backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java]: Yes, we can and I did now, thank you!

**Pull request merged**